ContactList/ContactList/UpdateController.swift

```diff
@@ -0,0 +1,391 @@
+    }
+    
+    func fillCellsArray (person: Person) -> [Presentation] {
+        let firstName = NSLocalizedString("FIRST_NAME_LABLE_TITLE", comment: "First name")
+        let lastName = NSLocalizedString("LAST_NAME_LABLE_TITLE", comment: "Last name")
+        let phone = NSLocalizedString("PHONE_LABLE_TITLE", comment: "Phone")
+        let email = NSLocalizedString("EMAIL_LABLE_TITLE", comment: "Email")
+        let birthday = NSLocalizedString("BIRTHDAY_LABLE_TITLE", comment: "Birthday")
+        let height = NSLocalizedString("HEIGHT_LABLE_TITLE", comment: "Height")
+        let note = NSLocalizedString("NOTE_LABLE_TITLE", comment: "Note")
+        let driverLicense = NSLocalizedString("DRIVER_LICENSE_LABLE_TITLE", comment: "Driver license")
+        
+        
+        var array: [Presentation] = [(Presentation(dataType: .image(person.image), cellType: .image)),
+                                 (Presentation(keyboardType: .default, placeholder: firstName, title: firstName, dataType: .text(person.firstName), cellType: .firstName, validationType: .forTextField(maxLength: 20))),
+                                 (Presentation(keyboardType: .default, placeholder: lastName, title: lastName, dataType: .text(person.lastName), cellType: .lastName, validationType: .forTextField(maxLength: 20))),
+                                 (Presentation(keyboardType: .numberPad, placeholder: phone, title: phone, dataType: .text(person.phoneNumber), cellType: .phoneNumber, validationType: .forPhoneNumber)),
+                                 (Presentation(keyboardType: .emailAddress, placeholder: email, title: email, dataType: .text(person.email), cellType: .email, validationType: .forEmail)),
+                                 (Presentation(placeholder: birthday, title: birthday, dataType: .date(person.birthday), cellType: .birthday)),
+                                 (Presentation(placeholder: height, title: height, dataType: .integer(person.height), cellType: .height)),
+                                 (Presentation(placeholder: note, title: note, dataType: .text(person.notes), cellType: .notes)),
+                                 (Presentation(title: driverLicense, dataType: .text(person.driverLicense), cellType: .driverLicenseSwitch))]
+        if !person.driverLicense.isEmpty {
+            array.append(getDriverLicenseNumberCell())
+        }
+        return array
+    }
+    
+    func getDriverLicenseNumberCell() -> Presentation {
+        let driverLicenseNumber = NSLocalizedString("DRIVER_LICENSE_NUMBER_LABLE_TITLE", comment: "Driver license number")
+        return Presentation(keyboardType: .default, placeholder: driverLicenseNumber, title: driverLicenseNumber, dataType: .text(currentPersonCopy.driverLicense), cellType: .driverLicense)
+    }
+    
+    func getCurentCell(at indexPath: IndexPath) -> UITableViewCell {
+        
+        switch indexPath.row {
```

Comment:
I'd recommend you not to use hardcoded index values, but rely entry in `cells`.
```
    let presentation = cells[indexPath.row]
    switch presentation.cellType {
        case .image:
        //dequeue and return image cell
        case .driverLicenseSwitch:
        //dequeue and return switch cell
        default:
        //dequeue and return text cell
    }
````